Header header
uint8 was_saved
uint32 saved_seq_num
